src/payment/cart/order-details/data/selectors.js

```diff
@@ -1,7 +1,7 @@
 import { createSelector, createStructuredSelector } from 'reselect';
 
-import { getModuleState } from '../../../common/utils';
-import { emailSelector } from '../../../common/selectors';
+import { getModuleState } from '../../../../common/utils';
```

Comment:
frontend-common can't get here soon enough... 🤦‍♂

Reply:
We could make an alias in webpack and clean this up but may be not worth it at the moment